cf-testd/cf-testd.c

```diff
@@ -136,6 +143,12 @@ CFTestD_Config *CFTestD_CheckOpts(int argc, char **argv)
         case 'I':
             LogSetGlobalLevel(LOG_LEVEL_INFO);
             break;
+        case 'l':
+            LoggingEnableTimestamps(true);
+            break;
+        case 'k':
+            config->key_file = xstrdup(optarg);
+            break;
         case 'p':
```

Comment:
swap 'l' and 'k' to put them in alpha order?

Reply:
yeap, good catch